src/FamilyHubs.Referral.Web/Pages/ProfessionalReferral/LocalOfferDetail.cshtml.cs

```diff
@@ -40,26 +40,30 @@ public async Task<IActionResult> OnGetAsync(string id, string serviceid)
         ReturnUrl = Request.Headers["Referer"].ToString();

```

Comment:
This is code that was already there, but if there is no Referer this will throw a `NullReferenceException`.
We need to do something sensible with the Back button when there is no referer.

Reply:
When I test the Back button it is fine event without a referrer